etc/config-template.toml

```diff
@@ -228,7 +228,7 @@
 # This value represents the maximum number of threads that will concurrently perform a
 # compaction job by breaking it into multiple, smaller ones that are run simultaneously.
 # Default: 1 (i.e. no subcompactions)
-# max-sub-compactions = 1
+max-sub-compactions = 2
```

Comment:
we also need to change the default value in Config structure.

Reply:
oh, it's better than change it here. So I will keep the line commented.